lib/generators/metka/strategies/materialized_view/templates/migration.rb.erb

```diff
@@ -5,41 +5,48 @@ class <%= @migration_class_name %> < ActiveRecord::Migration<%= ActiveRecord::VE
     execute <<-SQL
     CREATE OR REPLACE FUNCTION metka_refresh_<%= view_name %>_materialized_view() RETURNS trigger LANGUAGE plpgsql AS $$
       BEGIN
-        IF TG_OP = 'INSERT' AND NEW.<%= source_column_name %> IS NOT NULL THEN
+        IF TG_OP = 'INSERT' AND
+           (<%= source_columns.map { |column| "NEW.#{column} IS NOT NULL" }.join(' OR ') %>) THEN
           REFRESH MATERIALIZED VIEW CONCURRENTLY <%= view_name %>;
         END IF;
 
-        IF TG_OP = 'UPDATE' AND OLD.<%= source_column_name %> != NEW.<%= source_column_name %> THEN
+        IF TG_OP = 'UPDATE' AND
+           (<%= source_columns.map { |column| "OLD.#{column} IS DISTINCT FROM NEW.#{column}" }.join(' OR ') %>) THEN
```

Comment:
That's actually an important change, since before that `!=` operator wasn't interacting with NULLs correctly, so the materialized views wasn't refreshed on tagged column update with NULL value, while `IS DISTINCT FROM` does that correctly

Reply:
👍